config/initializers/db2fog.rb

```diff
@@ -0,0 +1,7 @@
+# See: https://github.com/yob/db2fog
+DB2Fog.config = {
+    :aws_access_key_id     => Spree::Config[:s3_access_key],
+    :aws_secret_access_key => Spree::Config[:s3_secret],
+    :directory             => Spree::Config[:s3_bucket],
```

Comment:
:alarm_clock: Does this mean we are uploading our database to the public bucket where our assets are stored? The reason why there were two config files is that we have the Spree config which is for assets (public bucket) and a second config (possibly a different access key) to upload backups to a private bucket. Maybe I missed something but this looks like we completely messed it up.

Reply:
👀 that means the release needs to be blocked: #3691

Reply:
I can confirm UK db backups are not public, but there are a number of issues here that need to be looked at. As far as I can tell they are pre-existing.